datahub-web-react/src/app/shared/time/__tests__/timeUtils.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { addInterval } from '../timeUtils';
+import { DateInterval } from '../../../../types.generated';
+
+describe('timeUtils', () => {
+    describe('addInterval', () => {
+        it('add date interval works correctly', () => {
+            const input = new Date(1677242304000);
+            const afterAdd = addInterval(1, input, DateInterval.Month);
+            const expected = new Date(1679661504000);
```

Comment:
This is failing for me because the result is an hour off in my timezone, I think because the next month in my timezone is an hour shorter due to daylight savings? The result I get is 1679657904000. Is the addInterval function calculating the interval in GMT time?

Reply:
https://github.com/datahub-project/datahub/pull/7484 Will see if we can reliably reproduce the failure in CI. Then we will work on fixing it. This should have been caught in the CI